File: beecrowd_exercicios_em_Python/ad-hoc/kermesse.py
def kermesse(l):
    for i in range(len(l)):
        if l[i] == i + 1:
            return l[i]


n = int(input())
i = 1
while n != 0:
    line = list(map(int, input().split()))

    print("Teste %d\n%d\n" % (i, kermesse(line)))

    n = int(input())

    i += 1

File: beecrowd_exercicios_em_Ruby/ad_hoc/kermesse.rb
def kermesse(lines)
  (0...lines.length).each do |x|
    return lines[x] if lines[x] == x + 1
  end
end

n = gets.to_i
i = 1
while n != 0

  lines = gets.split.map(&:to_i)

  puts "Teste #{i}\n#{kermesse(lines)}"
  puts ''
  i += 1

  n = gets.to_i

end
